### DISCUSSION TITLE
Can I just use go build to compile instead of bazel?

### DISCUSSION THREAD

**Discussion by wittwitt** [OffchainLabs/prysm#9313] [General]
Bazel broken my server GO environment  and I am not sure there are some necessary params of prysm

**Comment by prestonvanloon** [comment]
Yes, you can use pure go to build Prysm. Please read our rationale on bazel and this blurb about building with go vs bazel.

https://docs.prylabs.network/docs/reading/bazel#with-go

**Reply by wittwitt** [reply]
Thanks, I'll try .  Um , I think go build should be recommended first.
